keras/utils/vis_utils.py

```diff
@@ -32,7 +26,6 @@ def model_to_dot(model, show_shapes=False, show_layer_names=True):
     from ..layers.wrappers import Wrapper
     from ..models import Sequential
 
-    _check_pydot()
     dot = pydot.Dot()
```

Comment:
That's strange, it turns out no exception been raised in my computer. And I am pretty sure my keras version is up-to-date, I installed it from source code just now...